README.md

```diff
@@ -1,38 +1,36 @@
-# gitautosync
-
-Synchronizes a github repository with a local repository. Automatically deals with conflicts and produces useful output to stdout.
+# nbgitpuller
 
+One-way synchronization of a remote git repository to a local git repository,
+with automatic conflict resolution.
 
 # Installation
 
-Before installing, make sure you have a **Python 3** installation with
-the Jupyter environment installed.
-
-To install gitautosync, first install the package with `pip`. You should install
-the ``master`` branch of ``gitautosync``:
+There is no package on PyPI available right now. You can install directly from master:
 
     pip install git+https://github.com/data-8/gitautosync
 
-Next, enable the server extension with the following command:
-
-    jupyter serverextension enable --py nbgitautosync --sys-prefix
-
+You can then enable the serverextension
 
-# Usage
+    jupyter serverextension enable --py nbgitpuller --sys-prefix
 
-To use gitautosync, open the notebook. Depending whether you are working
-locally or via a JupyterHub, you'll use different syntax.
+# How it works
 
-If you're working off of a JupyterHub, use this syntax:
+nbgitpuller works by allowing you to construct a special URL, that when clicked
```

Comment:
nbgitpuller allows you to construct URL that points to a GitHub repository.
When it is clicked, `nbgitpuller` will pull for the contents of this repository
into the user's current folder within Jupyter, while rendering a nice status page.
This is especially useful when running on a JupyterHub, since it allows easy distribution
 of materials to users without requiring them to understand git.